### DISCUSSION TITLE
Alternate way to fund you, besides PayPal

### DISCUSSION THREAD

**Discussion by jeradGodsave** [devcompl/TTController#145] [Q&A]
Can I fund you using Venmo or some other way? 
Do you have a crowdfunding profile like Kickstarter?
I want to give $15 or $20, I just do not have Paypal and do not really trust it.

**Comment by MoshiMoshi0** [comment]
Thanks for wanting to support the project.
I do not have anything else other than the paypal link for donations, for now at least.
I could not find any donation site that is simple to setup, without requiring some id/bank confirmation, and has multiple ways for people to donate, card/paypal/bank transfer/single/monthly etc. I think closest is patreon but that is monthly only.
If you have any sites in mind let me know.

FYI i checked venmo and it seems to be owned by paypal, so I think you shouldnt have to make an account.
Thanks again!

**Comment by MoshiMoshi0** [comment]
You still have to make an paypal account if I request via email.
It's all good tho, I understand paypal is not ideal, if I figure out some additional way to support the project I will update you here.
Thanks again.
